### DISCUSSION TITLE
Not seeing any change on a virgin Win-10 install

### DISCUSSION THREAD

**Discussion by linuxgurugamer** [ChrisTitusTech/winutil#234] [Q&A]
Not sure if this is expected.

Have a laptop running Windows 10, fully updated. It's essentially a virgin install, not really used.

I used the script to install Ear Trumpet (love it, been using it for years), and just did the standard laptop tweaks. The number of processes didn't seem to change much

Task manager showed about 115-120 processes before and about the same after. I did try rebooting it, and didn't see any significant difference

Based on the video, I was expecting to see a 20-40% drop in the number of processes.

Lenovo Thinkpad T460s, 8 gig memory, i5-6200U cpu

**Comment by Lakshaya-wq** [comment]
@linuxgurugamer I'm also not sure why this happened to me as well on a Thinkpad T440, I saw 115-130 processes on stock windows install, and then running the tweak did not seem to cut them down. However, on my other laptop, with again a stock windows install, it cut down the processes to ~<70.
